data/tribes/buildings/productionsites/atlanteans/smelting_works/init.lua

```diff
@@ -55,13 +55,26 @@ tribes:new_productionsite_type {
          -- TRANSLATORS: Completed/Skipped/Did not start working because ...
          descname = _"working",
          actions = {
-            "call=smelt_iron",
+            "call=smelt_iron_1",
             "call=smelt_gold",
-            "call=smelt_iron",
+            "call=smelt_iron_2",
             "return=no_stats"
          }
       },
-      smelt_iron = {
+      smelt_iron_1 = {
```

Comment:
Please add a comment to each of the changed init.lua's explaining why there have to be two identical programs, so this won't be "optimized out" in the future by accident